### PR TITLE
Add problems

### PR DESCRIPTION
**Pull request by woojinnn**
Hello, CS220!

### CODE REVIEW

File: assets/why3/assignment05/pascal.mlw [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      for c = 1 to r-1 do
+        (* IMPORTANT: DON'T MODIFY THE ABOVE LINES *)
+        invariant { true (* TODO: Replace `true` with your solution *) }
+        (* IMPORTANT: DON'T MODIFY THE BELOW LINES *)
+        new_row[c] <- row[c-1] + row[c]
+      done;
+      row <- new_row
+    done;
+    row
+end
```
Comment (jeehoonkang) [inline assets/why3/assignment05/pascal.mlw:39]: - newline
- please check all files

File: Cargo.toml [hunk 2]
```diff
@@ -14,4 +14,10 @@ lazy_static = "1.4.0"
 pest = "2.5.1"
```
Comment (jeehoonkang) [inline Cargo.toml:14]: please upgrade all dependencies

**Review by jeehoonkang — CHANGES_REQUESTED** [review]
please update the private repo first and then migrate it to the public repo

File: assets/why3/assignment05/pascal.mlw (hunk 1 above)
Comment (AnHaechan) [reply assets/why3/assignment05/pascal.mlw:39]: For Why3, I removed trailing newlines at the end of files as you mentioned.
But for Rust, `cargo fmt` automatically adds a single newline at the end of files,
so I'll follow `cargo fmt` and send MR to private repo.

File: Cargo.toml (hunk 2 above)
Comment (AnHaechan) [reply Cargo.toml:14]: Will reflect on private repo first

**Pull request closed without merging**